src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp

```diff
@@ -73,8 +73,8 @@ class SubgraphBaseTest : public ov::test::TestsCommon {
 
     std::function<void(const std::exception& exp)> callback_exception = nullptr;
 
-    constexpr static const double disable_threshold = std::numeric_limits<double>::max();
-    double abs_threshold = disable_threshold, rel_threshold = disable_threshold;
+    constexpr static const double default_threshold = 0.01;
```

Comment:
default_threshold = 0; or should be changed default thresholds in compare() to 0.01